README.md

```diff
@@ -3,7 +3,7 @@ zend-diactoros
 
 [![Build Status](https://secure.travis-ci.org/zendframework/zend-diactoros.svg?branch=master)](https://secure.travis-ci.org/zendframework/zend-diactoros)
 
-> Diactoros: an epithet for Hermes, meaning literally, "the messenger."
+> Diactoros: an epithet for Hermes, meaning literally, "the messenger".
```

Comment:
American rules vs British rules.

Inside is american style and outside is british style (like most of the languages in the planet)